kernel/byterun/coq_interp.c

```diff
@@ -1308,11 +1320,13 @@ value coq_interprete
                     needs to be checked at this point */
         value divisor;
         divisor = *sp++;
-        if (uint63_eq0(divisor)) {
+        int b;
+        Uint63_eq0(b, divisor);
```

Comment:
Are we confident no allocation can kill `divisor` here? It might be better to adjust the stack later.